docs/usage.md

```diff
@@ -81,13 +167,58 @@ nextflow run nf-core/mhcquant \
   --input 'samplesheet.tsv' \
   --outdir <OUTDIR> \
   --fasta 'SWISSPROT_2020.fasta' \
-  <SEARCH PARAMS> \
   --peptide_min_length 8 \
-  --peptide_max_length 14 \
+  --peptide_max_length 12 \
   --ms2pip_model 'Immuno-HCD' \
   -profile docker
 ```
 
+### Example workflows
+
+**Basic identification workflow (default)**:
+```console
+nextflow run nf-core/mhcquant \
+  --input 'samplesheet.tsv' \
+  --fasta 'SWISSPROT_2020.fasta' \
+  --outdir results \
+  -profile docker
+```
+
+**Quantification workflow**:
+```console
+nextflow run nf-core/mhcquant \
+  --input 'samplesheet.tsv' \
+  --fasta 'SWISSPROT_2020.fasta' \
+  --quantify \
+  --outdir results \
+  -profile docker
+```
+
+**Global FDR with spectrum library generation**:
+```console
+nextflow run nf-core/mhcquant \
+  --input 'samplesheet.tsv' \
+  --fasta 'SWISSPROT_2020.fasta' \
+  --global_fdr \
+  --generate_speclib \
+  --outdir results \
+  -profile docker
+```
+
+**Complete workflow with all features**:
```

Comment:
Add to this run the addition of im2deep within the set of feature_generators in the --feature_generators flag